_layouts/post.html

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+---
+
+{% if page.image %}
+  {% unless page.image_hero == false %}
+    <div class="post-feature_image" style="background-image: url('{{ site.baseurl }}{{ page.image }}')"></div>
+  {% endunless  %}
+{% endif %}
+
+<div class="usa-grid">
+
+  <a class="back-blog link-arrow-left" href="{{ site.baseurl }}/blog/">Blog</a>
+  <article class="post" itemscope itemtype="http://schema.org/BlogPosting">
+    <header class="post-header">
+      <h1 class="post-title" itemprop="name headline">{{ page.title }}</h1>
+      <p class="post-byline">
+      {% if page.authors %}
```

Comment:
@gboone if you know how to do this, would you mind making the change on this branch?